modules/core/04-channel/keeper/packet.go

```diff
@@ -33,6 +33,13 @@ func (k Keeper) SendPacket(
 		return 0, errorsmod.Wrap(types.ErrChannelNotFound, sourceChannel)
 	}
 
+	if channel.FlushStatus != types.NOTINFLUSH {
+		return 0, errorsmod.Wrapf(
+			types.ErrInvalidChannelState,
+			"channel should not be in %s state", types.NOTINFLUSH.String(),
```

Comment:
minor preference for single line error return (I prefer to limit the amount of lines errors take when they aren't the most important check like proof verification, I would actually want to condense the != OPEN check eventually)